server/package.json

```diff
@@ -15,12 +15,14 @@
   },
   "homepage": "https://github.com/sb-js/socketio-react-chat#readme",
   "dependencies": {
+    "amqplib": "^0.5.5",
     "lodash": "^4.17.10",
     "openid-client": "^2.1.0",
     "shortid": "^2.2.11",
     "socket.io": "^2.1.1"
   },
   "devDependencies": {
+    "@types/amqplib": "^0.5.13",
```

Comment:
I added this library, but I didn't add any logic to the code yet, since I have got to learn a few things first. But from reading the gist is that amqplib is to be used to have a proper way to deal with the persistence of the data.